server/app/services/applicant/question/ApplicantQuestion.java

```diff
@@ -129,6 +129,14 @@ public String getQuestionText() {
     return repeatedEntity.map(r -> r.contextualize(text)).orElse(text);
   }
 
+  /**
+   * Get the question text that should be used for screen readers. We add a styled asterisk, which
```

Comment:
```suggestion
   * Returns the question text that should be used for screen readers.
   *
   * <p>We add a styled asterisk, which
```

Reply:
Gets is also correct, the phrasing just needs to read as if it was after "This method ..."